scripts/openshiftci-presubmit-all-tests.sh

```diff
@@ -2,63 +2,31 @@
 
 # fail if some commands fails
 set -e
+# Hide command that deals with secrets
+set +x
+if [[ -f $ODO_RABBITMQ_AMQP_URL ]]; then
+    export AMQP_URI=$(cat $ODO_RABBITMQ_AMQP_URL)
+fi
+if [[ -z $AMQP_URI ]]; then
+    echo "Please set AMQP_URI env with amqp uri or provide path of file containing it as ODO_RABBITMQ_AMQP_URL env"
+    exit 1
+fi
+export JOB_NAME="odo-pr-build"
+export REPO_URL="https://github.com/openshift/odo"
+export TARGET="$(jq .refs.pulls[0].number <<< $(echo $JOB_SPEC))"
+export CI_FIREWALL_VERSION="valpha"
+echo "Initiating"
 # show commands
 set -x
-
-ARCH=$(uname -m)
```

Comment:
Are these changes going to impact multi-arch team in any way? Have we communicated this to them?

Reply:
/cc @sarveshtamba

Reply:
These changes may impact P/Z. CC:- @zhengxiaomei123

Reply:
@sarveshtamba @zhengxiaomei123  letting you know so you are aware.
The idea here is to do the actual testing against internal resources like PSI. But exporting the logs here in a manner of speaking
It might be a good idea if we can hash this on a call. Let me know on slack